core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala

```diff
@@ -362,6 +364,12 @@ private[serializer] object KryoSerializer {
     classOf[StorageLevel],
     classOf[CompressedMapStatus],
     classOf[HighlyCompressedMapStatus],
+    classOf[RoaringBitmap],
+    classOf[RoaringArray],
+    classOf[RoaringArray.Element],
```

Comment:
Class RoaringArray.Element was protected or private and has been removed. Moreover, it is unclear to me why you even need to declare ``RoaringArray``, ``ArrayContainer`` and ``BitmapContainer``. Roaring bitmaps use their own serialization process that does not go through the standard Java serialization process.